cpp/models/ode_secirvvs/model.h

```diff
@@ -32,9 +32,55 @@ namespace mio
 {
 namespace osecirvvs
 {
-class Model : public CompartmentalModel<InfectionState, Populations<AgeGroup, InfectionState>, Parameters>
+template <class I = InfectionState>
```

Comment:
"I" is only used locally to abbreviate the definitions of the Flows for each model.
In my opinion, using "I" instead of "InfectionState" or "InfSt" makes these a lot more readable, because the compartment appears thrice with each Flow.